addons/project_mail_plugin/controllers/project_client.py

```diff
@@ -0,0 +1,41 @@
+
+from odoo import http
+from odoo.http import request
+from odoo.tools import html2plaintext
+
+
+class ProjectClient(http.Controller):
+    @http.route('/mail_plugin/project/search', type='json', auth='outlook', cors="*")
+    def projects_search(self, search_term, limit=5):
+
```

Comment:
I'm guessing this is used on the Outlook side when searching projects in some kind of search bar?
Could be nice to give some context as to how it's used in the docstring.

Reply:
Done, docstring added.